client/components/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@automattic/simple-components",
+  "version": "0.1.0-alpha.1",
+  "description": "React components, as used on WordPress.com",
+  "scripts": {
+    "build": "babel ./index.js --out-dir cjs && babel ./button --out-dir cjs/button && babel ./main --out-dir cjs/main",
```

Comment:
We might want to consider moving components that are supposed to go into `simple-components` into a separate subfolder, e.g. `client/components/simple`. This is going to save us a lot of hassle.